### DISCUSSION TITLE
Active User & Gaming

### DISCUSSION THREAD

**Discussion by kin1076** [kinfoundation/kin-rfcs#117] [_ Archive - KRE 3.1 Discussion]
One quick question... What qualifies as an active user?  Is it still 3 spends in 30 days or is it one spend of 1,000 Kin in 30 days?  

The main question is how much governance can KF have over these spends/earns?  Is it possible to deem certain transactions as "meaningless" in some kind of way?  There are many ways to make meaningless spends that can still end up gaming this system.  While I understand that decentralization is a goal, KF has never suggested this includes the governance of the KRE.  So control of THIS particular part of the project is necessary to make things proper.  

I don't know the exact definitions, parameters, etc on what this type of control would entail, but I don't think a system will be  "gameproof" for lack of a better made up word, unless there is more monitoring and control of what is deemed an appropriate spend.  Ironically a tip is an appropriate spend (coming from the mouth of Ted himself) and yet there are a few much more "valuable" spends that are completely meaningless in the ecosystem as we speak.  

The comment that gaming the system with 1 Kin spends is easy, does not mean that gaming this system is any harder.  It is just as easy no matter how much that Kin spend minimum ends up.  What matters is the spend's meaning.  Anyone can put in a button that says spend 100,000 Kin and earn 100,001 Kin back.  

The intention of this KRE is wonderful.  The execution falls short if one of the goals is to reduce gaming.  Other parts of it definitely succeed in creating a more value driven economy.  I'm just curious to know whether it is possible to create a comprehensive list of meaningful spends allowed in apps, using the current apps in the world.  There are enough out there to suggest you can easily do this with some time and effort (I'm sure community members would help).  If somehow someone comes up with a unique spend they can simply request a review and then it can be included.  

Any thoughts would be great.  Thanks.

**Comment by skyf0xx** [comment]
An Active User is a user who has spent >= 3 times, with each spend being at least 1,000 KIN.

We actually believe spends will go well above this amount, just based on how KRE 3.1 incentive model works, but we wanted to set a floor for spends. (A few cents).

In terms of what is considered a valid spend, we have a [checklist](https://developer.kin.org/docs/transaction-guide/) that the developer must follow. The checklist is intentionally generalized to give developers as much room to innovate as possible.

**Comment by skyf0xx** [comment]
In terms of gaming, no system is 100% free of vectors. We account for this by picking the least gameable incentives we can find. For example, AUB is harder to game than rewarding based on transaction counts.

On top of this, we also monitor and regularly contact apps if we detect any unusual behavior.  Anyone gaming AUB beyond a threshold is very easy to find.

This is sometimes caused by bots by harvesters attacking some apps. Bot transactions are also fairly obvious if you know what to look for.

We work with devs to help resolve this and this normally works out quite smoothly.
